src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -1274,7 +1423,8 @@ await DownloadFileAsync(file.FilePath, destination, cancellationToken)
             InlineKeyboardMarkup replyMarkup = default,
             CancellationToken cancellationToken = default,
             bool sendPhoneNumberToProvider = default,
-            bool sendEmailToProvider = default
+            bool sendEmailToProvider = default,
+            bool allowSendingWithoutReply = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.